tests/network/flat_overlay/utils.py

```
@@ -20,6 +21,14 @@
 LOGGER = logging.getLogger(__name__)
 
 
+def restart_ovnkube_node_daemonset() -> None:
+    """Restarts the ovnkube-node DaemonSet and waits for the rollout to complete."""
+    subprocess.check_call(shlex.split("oc rollout restart daemonset/ovnkube-node -n openshift-ovn-kubernetes"))
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)